examples/asr/asr_cache_aware_streaming/speech_to_text_cache_aware_streaming_infer.py

```diff
@@ -243,6 +251,9 @@ def main():
     parser.add_argument(
         "--output_path", type=str, help="path to output file when manifest is used as input", default=None
     )
+    parser.add_argument(
+        "--pad-and-drop-preencoded", action="store_true", help="pad first audio chunk and always drop preencoded"
+    )
```

Comment:
By default in runs as in main.  you can set `--pad-and-drop-preencoded` pad the first chunk and always drop

Reply:
Let's make the comment with more detail, something like this:
"Enables padding the audio input and then dropping the extra steps after the pre-encoding for the first step. It makes the outputs of the downsampling exactly as the offline mode for some techniques like striding."